src/main/java/core/basesyntax/service/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.Colors;
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
+
+    public String getRandomColor() {
+        return Colors.values()[randomColor.nextInt(Colors.values().length)].name();
```

Comment:
5 method calls in one line is too much and hard to read - better split it into 2 lines.